### PR TITLE
fix: add subhadeepbanerjee2003

### LINKED ISSUE #1899: [PROFILE ADDITION] <subhadeepbanerjee2003>
### Profile screenshot

- **SCREENSHOT**

Screenshot from 2023-08-23 00-29-20

### PR DESCRIPTION
**Pull request by subhadeepbanerjee2003**
closes #1899 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes
```
{
  "image": "https://user-images.githubusercontent.com/116381157/262457570-69139248-1a35-416c-b2f5-40a19cf9969c.png",
  "issueId": 1899,
  "name": "Subhadeep Banerjee",
  "username": "subhadeepbanerjee2003"
}
```

### CODE REVIEW

**Comment by subhadeepbanerjee2003** [comment]
**I hope maintainers will love my work as i have done the merge request with in detail description :)**

File: profiles/subhadeepbanerjee2003.json
Comment (schmelto) [inline profiles/subhadeepbanerjee2003.json:8]:
```suggestion
}
```

**Review by subhadeepbanerjee2003 — COMMENTED** [review]
Changes done

File: profiles/subhadeepbanerjee2003.json
Comment (schmelto) [inline profiles/subhadeepbanerjee2003.json:8]:
```suggestion
```

**Review by schmelto — APPROVED** [review]
Your suggested changes looking good to me. Thanks for your contribution!

---
**Summary:** :ok_hand: This code looks good

**Comment by subhadeepbanerjee2003** [comment]
Is my pr merged? @schmelto

**Comment by schmelto** [comment]
As per our merging process, we require approvals from at least two maintainers before proceeding. Since we currently have only one approval, kindly wait for another maintainer to review and provide their approval for this Pull Request.

For reference take a look [here](https://eddiehubcommunity.github.io/support/#/moderators/github?id=when-to-merge-prs).

---
**Summary:** 👍🏼 needs another approval

**Comment by subhadeepbanerjee2003** [comment]
@schmelto should I tag other maintainer?

**Comment by schmelto** [comment]
> @schmelto should I tag other maintainer?

no need for that I think

**Comment by subhadeepbanerjee2003** [comment]
@eddiejaoude @techhead @schmelto @stemount please merge it after reviewing

**Pull request merged**
